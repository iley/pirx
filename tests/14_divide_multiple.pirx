func main() {
  var result: int;
  result = 24 / 4 / 2;
  printf("24 / 4 / 2 = %d\n", result);
  result = 8 * 3 / 2;
  printf("8 * 3 / 2 = %d\n", result);
  result = 1 + 12 / 3 - 2;
  printf("1 + 12 / 3 - 2 = %d\n", result);
  return 0;
} 