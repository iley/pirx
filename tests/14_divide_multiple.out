24 / 4 / 2 = 3
8 * 3 / 2 = 12
1 + 12 / 3 - 2 = 3
